g2p/cli.py

```diff
@@ -353,7 +353,7 @@ def generate_mapping(  # noqa: C901
             except MappingMissing as e:
                 raise click.BadParameter(
                     f'Cannot find IPA mapping from "{in_lang}" to "{out_lang}": {e}',
-                    param_hint=["IN_LANG", "OUT_LANG"],
+                    param_hint="'IN_LANG' 'OUT_LANG'",
```

Comment:
Wow, I have no idea how to even trigger this exception!

Reply:
Ah, found it. You get it from
`g2p generate-mapping --ipa kwk-boas fra-ipa`
The old code generated
> Error: Invalid value for 'IN_LANG' / 'OUT_LANG': ...

the new code generates
> Error: Invalid value for 'IN_LANG' 'OUT_LANG': ...

I guess the param_hint typing lies, and is in fact more permission that it declares. I kinda like the slash, or even better would be `"'IN_LANG' & 'OUT_LANG'"` because it's the pair of values together that's invalid, not the individual values.

But that's just nitpicking, I know.

Reply:
oh! thanks, I actually don't know how it's supposed to be formatted.  It's the type signature that's wrong, not the documentation... so we could restore the old code and just put a 'type: ignore' on it and file a bug against click.

Reply:
Okay, issue filed with click (linking it here): https://github.com/pallets/click/issues/2777